db/schema.rb

```diff
@@ -152,10 +152,10 @@
     t.boolean "open_banking_consent"
     t.datetime "open_banking_consent_choice_at"
     t.string "own_home"
-    t.decimal "property_value", precision: 10, scale: 2
     t.decimal "percentage_home"
-    t.string "shared_ownership"
+    t.decimal "property_value", precision: 10, scale: 2
     t.decimal "outstanding_mortgage_amount"
+    t.string "shared_ownership"
```

Comment:
No big deal but ideally your PR shouldn't change the order of columns.
I think whenever you run a migration, `schema.rb` is modified to match the order of columns in your DB.
The order of columns in your local DB is different probably because you were testing some not merged PRs.
You can reset your DB to match `schema.rb` with the command `rails db:schema:load`